test/e2e/federated-service.go

```diff
@@ -56,7 +56,7 @@ const (
 	FederatedDNS1123Regexp = `([a-z0-9]([-a-z0-9]*[a-z0-9])?\.)*([a-z0-9]([-a-z0-9]*[a-z0-9])?)`
 )
 
-var _ = framework.KubeDescribe("Service [Feature:Federation]", func() {
+var _ = framework.KubeDescribe("Federated Services", func() {
```

Comment:
Whats the advantage of moving?
Does that mean that BeforeEach and AfterEach will still run when we are not running Feature:Federation. Do we want that?